interface/drugs/drug_inventory.php

```diff
@@ -134,31 +134,33 @@ function mapToTable($row)
 <title><?php echo xlt('Drug Inventory'); ?></title>
 
 <style>
-a, a:visited, a:hover { color:#0000cc; }
+a, a:visited, a:hover {
```

Comment:
@bradymiller why do we have this set here?

Reply:
not sure. looks like it has been in this script since the beginning of time though.